app/models/case_contact.rb

```diff
@@ -279,6 +279,12 @@ def self.case_hash_from_cases(cases)
     end
   end
 
+  def form_steps
```

Comment:
If we're gonna do this pattern, then let's future proof the app a bit. There are a few calls to `CaseContact::FORM_STEPS` in the app. Replace those with this method.

1. `CaseContactsController:58`
2. `CaseContactsController:65`
3. `CaseContact:38`